TestStack.Dossier/TestDataBuilder.cs

```diff
@@ -144,9 +155,10 @@ public TValue Get<TValue>(Expression<Func<TObject, TValue>> property)
         /// <returns></returns>
         public object Get(Type type, string propertyName)
         {
-            if (!Has(propertyName))
-                return Any.Get(type, propertyName);
-            return _properties[propertyName];
+            Func<object> factory;
+            if (_properties.TryGetValue(propertyName, out factory)) return factory();
```

Comment:
I'd recommend starting a document with this case, and add to it in the future.  It's hard to pick up on what the coding style rules of a project are just from the rest of the code, especially for a new contributor.  If you want, I'll include it in my pull request.